conformance/operator_test.go

```diff
@@ -265,6 +267,114 @@ var _ = Describe("operator", func() {
+						NicSelector: sriovv1.SriovNetworkNicSelector{
+							PfNames: []string{intf.Name},
+						},
+						DeviceType: "netdevice",
+					},
+				}
+
+				err = clients.Create(context.Background(), nodePolicy)
+				Expect(err).ToNot(HaveOccurred())
+
+				Eventually(func() bool {
+					stable, err := cluster.SriovStable(operatorNamespace, clients)
+					Expect(err).ToNot(HaveOccurred())
+					return stable
+				}, 5*time.Minute, 1*time.Second).Should(Equal(true))
+
+				Eventually(func() int64 {
+					testedNode, err := clients.Nodes().Get(node, metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+					resNum, _ := testedNode.Status.Capacity["openshift.io/apivolresource"]
+					capacity, _ := resNum.AsInt64()
+					return capacity
+				}, 3*time.Minute, time.Second).Should(Equal(int64(5)))
+
+				sriovNetwork := &sriovv1.SriovNetwork{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "apivolnetwork",
+						Namespace: operatorNamespace,
+					},
+					Spec: sriovv1.SriovNetworkSpec{
+						ResourceName:     "apivolresource",
+						IPAM:             `{"type":"host-local","subnet":"10.10.10.0/24","rangeStart":"10.10.10.171","rangeEnd":"10.10.10.181","routes":[{"dst":"0.0.0.0/0"}],"gateway":"10.10.10.1"}`,
+						NetworkNamespace: namespaces.Test,
+					}}
+				err = clients.Create(context.Background(), sriovNetwork)
+				Expect(err).ToNot(HaveOccurred())
+
+				pod := &corev1.Pod{
+					ObjectMeta: metav1.ObjectMeta{
+						GenerateName: "apivolpod",
+						Namespace:    namespaces.Test,
+						Annotations:  map[string]string{"k8s.v1.cni.cncf.io/networks": "apivolnetwork"},
+					},
+					Spec: corev1.PodSpec{
+						TerminationGracePeriodSeconds: pointer.Int64Ptr(0),
+						Containers: []corev1.Container{
+							{
+								Name:    "test",
+								Image:   "fedora:31",
+								Command: []string{"/bin/bash", "-c", "sleep INF"},
+							}}},
+				}
+
+				created, err := clients.Pods(namespaces.Test).Create(pod)
+				Expect(err).ToNot(HaveOccurred())
+
+				var runningPod *corev1.Pod
+				Eventually(func() corev1.PodPhase {
+					runningPod, err = clients.Pods(namespaces.Test).Get(created.Name, metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+					return runningPod.Status.Phase
+				}, 3*time.Minute, time.Second).Should(Equal(corev1.PodRunning))
+
+				var downwardVolume *corev1.Volume
+				for _, v := range runningPod.Spec.Volumes {
+					if v.Name == "podnetinfo" {
+						downwardVolume = v.DeepCopy()
+					}
+				}
+
+				Expect(downwardVolume).ToNot(BeNil(), "Downward volume not found")
+				Expect(downwardVolume.DownwardAPI).ToNot(BeNil(), "Downward api not found in volume")
+				Expect(downwardVolume.DownwardAPI.Items).To(SatisfyAll(
+					ContainElement(corev1.DownwardAPIVolumeFile{
+						Path: "labels",
+						FieldRef: &corev1.ObjectFieldSelector{
+							APIVersion: "v1",
+							FieldPath:  "metadata.labels",
+						},
+					}), ContainElement(corev1.DownwardAPIVolumeFile{
+						Path: "labels",
```

Comment:
I think this one should be `annotations`

Reply:
Thanks for catching it!